app/ProjectAanvraag/ProjectAanvraagClient.php

```diff
@@ -6,24 +6,30 @@
 
 use App\Json;
 use App\ProjectAanvraag\Requests\CreateWidgetRequest;
-use GuzzleHttp\Client;
 use GuzzleHttp\Psr7\Request;
+use Psr\Http\Client\ClientExceptionInterface;
 use Psr\Http\Client\ClientInterface;
 use Psr\Log\LoggerInterface;
 
+use JsonException;
+
 final readonly class ProjectAanvraagClient
 {
     public function __construct(
         private LoggerInterface  $logger,
-        private ?ClientInterface $client = null
+        private ClientInterface $httpClient,
     ) {
     }
 
+    /**
+     * @throws ClientExceptionInterface|JsonException
+     */
     public function createWidget(CreateWidgetRequest $createWidgetRequest): void
     {
+        $baseUri = ProjectAanvraagUrl::getStatusBaseUri($createWidgetRequest->status);
```

Comment:
Now that I read this method name I think it is wrong and should be something like `getBaseUrlForIntegrationStatus`

What it does is get the test or production URL based on the status of the integration.

Reply:
Yes indeed

```php
$stage = $integrationStatus === IntegrationStatus::Active ? 'live' : 'test';
return config('project_aanvraag.base_uri.' . $stage);
```

Reply:
I'll do the rename

Reply:
[Rename getBaseUrlForIntegrationStatus](https://github.com/cultuurnet/publiq-platform/pull/906/commits/b0402d9be68f315bd5f54de56c74ee3d275ed0b2)